src/routers/webhooks.ts

```diff
@@ -1,8 +1,24 @@
+interface NationBuilderPerson {
+  email: string;
+  firstName: string;
+  phone: string;
+}
+
 import express from 'express';
 const router = express.Router();
 
-router.use('/', (_, res) => {
-  res.send('webhook route');
+router.use('/', (_, res, next) => {
+  next();
+});
+
+router.post('/nationbuilder/personCreated', function(req, res) {
+  const nationBuilderPerson: NationBuilderPerson = {
+    email: req.body.payload.person.email,
+    firstName: req.body.payload.person.first_name,
+    phone: req.body.payload.person.phone,
```

Comment:
nit: destructure this above:

```
const { email, first_name, phone } = req.body.payload.person;
```

Reply:
Cool! Attaching the link you sent me for documentation:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment